cms/templates/settings.html

```diff
@@ -308,6 +308,21 @@ <h2 class="title-2">${_("Requirements")}</h2>
                   <input type="text" class="short time" id="course-effort" placeholder="HH:MM" />
                   <span class="tip tip-inline">${_("Time spent on all course work")}</span>
                 </li>
+                % if settings.FEATURES.get('ENABLE_PREREQUISITE_COURSES'):
+                <li class="field field-select" id="field-pre-requisite-course">
+                  <form action="#" class="pre-requisite-course-form" method="post">
+                    <label for="pre-requisite-course">${_("Prerequisite Course")}</label>
+                    <select class="input" id="pre-requisite-course">
+                        <option value="">${_("None")}</option>
+                        % for course_info in sorted(pre_requisite_courses, key=lambda s: s['display_name'].lower() if s['display_name'] is not None else ''):
```

Comment:
I just noticed that we're showing the display name in the drop down list. I'm wondering that if there are two runs of the same course, it's possible that authors will keep the same course display name between runs and this may cause a situation where the DDL is ambiguous.

It's probably not important now, but maybe something to do as a follow-up release

@marcotuts?